uw-frame-components/portal/notifications/controllers.js

```diff
@@ -93,6 +95,19 @@ define(['angular'], function(angular) {
     $scope.switch = function(mode) {
       $scope.mode = mode;
     }
+    
+    $scope.priorityNotifications = [];
+    notificationsService.getAllNotifications().then(function(data) {
+      var notifications = data;
+      var priorityNotifications = [];
+      for( var i=0; i < notifications.length; i++ ) {
+        if (notifications[i].priority === true) {
+          $scope.priorityNotifications.push(notifications[i]);
+          $scope.hasPriorityNotifications = true;
+        };
+      };
+    }, function(data) {});
+    
```

Comment:
This seems redundant, couldn't we just filter on the current list of notifications?